pkg/newlog/cmd/newlogd.go

```diff
@@ -63,22 +65,28 @@ const (
 
 	minSpaceCleanupMB uint64 = 100 // start to cleanup if space in /persist is less than 100M
 
+	maxToSendMbytes uint32 = 0xFFFFFFFF // unlimited for default log remains on disk
+
+	maxSentFileKeep int = 1000 // keep up to 1000 already uploaded gzip files in circular queue
```

Comment:
Why do we need to limit the number of files if we limit the Mbytes? Can lead to surprises for the user.

Reply:
@eriknordmark this limit is for circular buffer directory, not the quota limit

Reply:
But the user is expresssing a disk size limit, not a limit on the number of files.

Reply:
changed. only use size now.